### PR TITLE
YLP-2188 Sort patients by column on patient list

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Review by TimCanu — COMMENTED** [review]
I have not tested it yet

**Review by TimCanu — APPROVED** [review]

File: packages/yourloops/components/patient/patient-info.tsx
Comment (fatb38) [inline packages/yourloops/components/patient/patient-info.tsx:1]: It was some dead code ?

File: packages/yourloops/test/integration/assert/patient-list.ts [hunk 1]
```diff
@@ -580,3 +586,45 @@ export const checkPatientListTooltips = async (): Promise<void> => {
...
+  await userEvent.click(sortButton)
+
+  const allRowsAfterSecondSort = within(dataGridRows).getAllByRole('row')
+  expect(allRowsAfterSecondSort[1]).toHaveTextContent(monitoredPatient.profile.fullName)
+  expect(allRowsAfterSecondSort[2]).toHaveTextContent(monitoredPatientTwo.profile.fullName)
+  expect(allRowsAfterSecondSort[3]).toHaveTextContent(monitoredPatientWithMmol.profile.fullName)
+  expect(allRowsAfterSecondSort[4]).toHaveTextContent(unmonitoredPatient.profile.fullName)
+}
+
+export const checkMonitoringAlertsIconsInactiveForFirstPatient = async (): Promise<void> => {
```
Comment (fatb38) [inline packages/yourloops/test/integration/assert/patient-list.ts:618]: Could it be more generic ? Why do you have to test it on the first patient ? Can we have a test on any patient and not particulary the first one of the list ?

File: packages/yourloops/components/patient/patient-info.tsx
Comment (celia-rouis) [reply packages/yourloops/components/patient/patient-info.tsx:1]: Yes!

File: packages/yourloops/test/integration/assert/patient-list.ts (hunk 1 above)
Comment (TimCanu) [reply packages/yourloops/test/integration/assert/patient-list.ts:618]: I just want to point out that in most cases, this is not helpful to have generic code in test :wink:

**Pull request merged**